.github/dependabot.yml

```diff
@@ -45,7 +45,7 @@ updates:
     # Spring Cloud dependencies
     # spring-cloud-dependencies-parent should be eventually removed per #1294
     - dependency-name: "org.springframework.cloud:spring-cloud-*"
-      versions: ["2022.x", "2023.x", "4.x"]
+      versions: ["2022.x", "2023.x", "2024.x", "4.x"]
```

Comment:
"4.x" looks odd to me?

Reply:
Indeed, they only use calendar versioning. Removing